graphics/opencv/Portfile

```diff
@@ -53,24 +55,29 @@ platform macosx {
 
 worksrcdir          opencv-${version}
 
-conflicts           opencv4
-
 compiler.cxx_standard   2011
-compiler.blacklist-append cc *gcc* {clang < 800} {macports-clang-3.[0-9]} macports-clang-4.0
+
+compiler.blacklist-append \
+                    cc \
+                    *gcc* \
+                    {clang < 800} \
+                    {macports-clang-3.[0-9]} \
+                    macports-clang-4.0
 
 depends_build-append \
                     port:pkgconfig
 
-depends_lib-append  port:zlib \
+depends_lib-append  \
+                    port:ffmpeg \
+                    port:zlib \
                     port:bzip2 \
                     port:libpng \
                     port:jpeg \
-                    port:jasper \
+                    port:openjpeg \
```

Comment:
why would we want to depends_lib both port:jpeg and port:openjpeg? 

Is that right? Seems weird.